tasks/plugins.yml

```diff
@@ -27,3 +27,19 @@
     creates=/var/lib/jenkins/plugins/{{ item }}.jpi
   with_items: jenkins_plugins
   notify: restart jenkins
+
+- name: Get Plugins to update
+  # plugins which have an update show the newest available version in () at the right
```

Comment:
Comments should go above the `name` line, and I'd prefer a style that's just one proper sentence, wrap at 80 cols, e.g.:

```
  # Plugins which have an update show the newest available version in () at the
  # right, e.g. `javadoc    Javadoc Plugin    1.1 (1.3)`.
```

Can you update the comment to match?